component/upgrade/patch.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+#Patch CRDS so that ArgoCD does not delete them during upgrade
+for crd in $CRDS_TO_PATCH; do
+	if ! kubectl get crd "${crd}" >/dev/null 2>&1; then
+		echo >&2 "WARNING: Skipping 'blah': not found."
```

Comment:
I don't think we want a literal `'blah'` in the error here:

```suggestion
		echo >&2 "WARNING: Skipping '${crd}': not found."
```

Reply:
Indeed